### PR TITLE
fix lastEventId tracking

### PR DESCRIPTION
**Pull request by michalskrivanek**
- core: remove redundant set
- core: fix lastEventId tracking

### CODE REVIEW

**Review by mnecas — APPROVED** [review]
lgtm

File: backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerClient.java [hunk 1]
```diff
@@ -53,23 +52,18 @@ public AnsibleRunnerClient() {
...
-                res = Files.readString(lastEventFile.toPath());
-            } catch (IOException e) {
-                return false;
-            }
-        return res.contains("playbook_on_stats");
+    public Boolean playHasEnded(String uuid, int lastEventId) {
+        // get current status, but new events might have appeared since last processing
+        PlaybookStatus currentPlaybookStatus = getPlaybookStatus(uuid);
+        String msg = currentPlaybookStatus.getMsg();
+        return !msg.equalsIgnoreCase("running") && lastEventId <= getTotalEvents(uuid);
```
Comment (mnecas) [inline backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerClient.java:59]: Shouldn't this be `lastEventId == getTotalEvents(uuid)` in the case when the playbook finish and the msg is no longer running we still process all events until the last one?

Comment (michalskrivanek) [reply backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerClient.java:59]: oh sure, the other way around:)
&& !(lastEventId < getTotalEvents(uuid))

Comment (mnecas) [reply backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerClient.java:59]: uhhh the negative does not look best... if you don't like the `==` how about the `>=` ?

**Review by mnecas — APPROVED** [review]

**Pull request merged**